src/EditorFeatures/CSharp/ChangeSignature/CSharpChangeSignatureCommandHandler.cs

```diff
@@ -13,7 +14,8 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.ChangeSignature
     internal class CSharpChangeSignatureCommandHandler : AbstractChangeSignatureCommandHandler
     {
         [ImportingConstructor]
-        public CSharpChangeSignatureCommandHandler()
+        public CSharpChangeSignatureCommandHandler(IThreadingContext threadingContext) :
```

Comment:
Per @CyrusNajmabadi's other PR, we're putting : before the base.

Reply:
sure